build.sbt

```diff
@@ -120,45 +120,44 @@ lazy val coreDependencies = (providedSparkDependencies ++ testCoreDependencies +
   "org.slf4j" % "slf4j-log4j12" % "1.7.25",
   "org.jdbi" % "jdbi" % "2.63.1",
   // Exclude extraneous GATK dependencies
-  ("org.broadinstitute" % "gatk" % "4.0.11.0")
+  ("org.broadinstitute" % "gatk" % "4.1.6.0")
```

Comment:
What's the net change on transitive dependencies?

Reply:
I added the diff to the PR description - it looks fairly reasonable to me.

Reply:
To get the transitive dependency list on sbt 1.3.0, I also updated `sbt-dependency-graph`.